File: apps/selvbetjening-ui/src/types/person.ts
export interface IInnloggetBruker {
    fornavn?: string
    etternavn?: string
    foedselsnummer?: string
    foedselsaar?: number
    foedselsdato?: Date | string
}

export enum Alder {
    ATTEN_TIL_SEKSTI_EN,
    SEKSTI_TO_TIL_SEKSTI_SEKS,
    SEKSTI_SYV,
    IKKE_GYLDIG,
}

File: apps/selvbetjening-backend/src/main/kotlin/person/Person.kt
package no.nav.etterlatte.person

import no.nav.etterlatte.libs.common.person.Foedselsnummer

data class Person(
    val fornavn: String,
    val etternavn: String,
    val foedselsnummer: Foedselsnummer,
    val foedselsaar: Int? = null,
    val foedselsdato: String? = null,
)